docs/api-reference/layers/geojson-layer.md

```diff
@@ -84,50 +85,80 @@ The `GeoJSONLayer` accepts any of the following formats passed to the `data` pro
 
 * A valid GeoJSON `FeatureCollection`, `Feature`, `Geometry` or `GeometryCollection` object.
 * An array of GeoJSON `Feature` objects.
+* An URL or Promise that resolves to the above formats.
+* loaders.gl's [flat GeoJSON format](https://loaders.gl/modules/gis/docs/api-reference/geojson-to-binary).
```

Comment:
It's called binary format in other places, for example in MVTLayer prop. I'd replace `flat GeoJSON format` for `GeoJSON binary format`